agent/acl_test.go

```diff
@@ -91,7 +91,7 @@ func NewTestACLAgent(t *testing.T, name string, hcl string, resolveAuthz authzRe
 
 func (a *TestACLAgent) ResolveToken(secretID string) (acl.Authorizer, error) {
```

Comment:
Does this need to be renamed to `ResolveTokenSecret` as well?

Reply:
This ties into the server struct in xds code, which would make more sweeping changes that I'd like to consider outside the scope of this PR
https://github.com/hashicorp/consul/blob/674c5570b6c2d8f0e00972bc130efe4626bed2d7/agent/xds/server.go#L108

Reply:
done!